modules/apps/27-interchain-accounts/types/errors.go

```diff
@@ -5,13 +5,14 @@ import (
 )
 
 var (
-	ErrUnknownPacketData     = sdkerrors.Register(ModuleName, 1, "Unknown packet data")
```

Comment:
SDK will treat any error code with `1` as an internal error and discard the error message